Добавлена функция считывания уровня освещенности. Closes #5

Timofey-arch opened the pull request.

Timofey-arch committed:
Добавлена функция считывания уровня освещенности #5

redb0 commented:
По реализации есть несколько замечаний:
- Функция `getLux` не возвращает результат измерения освещенности. Кроме этого эта функция не должна ничего выводить через `Serial` и вызывать `delay`. Все эти действия должны производиться все ее, например, в `loop`.
- Сам проект должен находиться в файле с расширением `cpp`, а не `ino`.
- Директория `lib` предназначена для библиотек, а не для основного кода. Весь основной код должен быть размещен в директории `growbox_sketch/src`, например в файле `main.cpp`. Код, относящийся к датчикам, лучше разместить в отдельном файле, который подключается в `main.cpp` через `include` и где будут использоваться эти функции.

Timofey-arch committed:
Изменена функция считывания уровня освещенности

Вынесен весь лишний код из функции в loop
Изменен формат файла на cpp
Изменено местоположение файла

redb0 commented:
Почему функция возвращает `int`, хотя переменная `lux` типа `uint16_t`? 

Нужно исправить сбившуюся кодировку в функции `loop` :moyai:

И нужно придерживаться правил контрибьюта https://github.com/Electro98/cyber-growbox-2077/blob/main/CONTRIBUTING.md

Timofey-arch committed:
:wrench: И снова изменена функция считывания уровня освещенности #5

Исправлена сбитая кодировка
Теперь функция возвращает uint16_t
